src/Components/AddComment.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { useEffect, useState, useContext } from "react";
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import { UserContext } from "../../context/userContext";
+import { postNewComment } from "../api";
+import { useParams } from "react-router-dom";
+import CommentCard from "./CommentCard";
+
+
+function AddComment () {
+    const {id} = useParams();
+    const { user:{username, name, avatar_url} } = useContext(UserContext);
+    const [commentBody, setCommentBody] = useState("");
+    const [placholderBody, setPlaceholderBody] = useState("Insert comment...")
+    const [message, setMessage] = useState("Post a Comment")
+
+    function handleClick (event) {
+        event.preventDefault()
+
+        if (commentBody !== "") {
```

Comment:
This if statement does the job to stop an empty message being sent, but it's not clear to the user. You could use something like required on the input or a piece of state to tell the user why it's not posted!